bigbluebutton-html5/imports/utils/visibilityEvent.js

```diff
@@ -9,14 +11,25 @@ export default class VisibilityEvent {
     this._handleVisibilityChange = this._handleVisibilityChange.bind(this);
 
     this._registerVisibilityEvent();
+
+    this._onHiddenTimeout = null;
   }
 
   onVisible (f) {
-    this._onVisible = f;
+    this._onVisible = () => {
+      if (!this._isHidden()) {
+        f();
```

Comment:
Please change `f` for `callback` or something more meaningful

Reply:
It's funny, but there seem to be cases where the event fires and immediately the window is visible again, e.g. dragging a tab and separating it into a new window. This caused the code to not work properly and I check again for visibility just in case.